goshimmer_install.sh

```diff
@@ -467,8 +568,25 @@ function check_arch() {
     fi
 }
 
+function check_total_ram_min_kb() {
+    MIN_RAM=$1
+    MEM_TOTAL=$(grep ^MemTotal /proc/meminfo | awk {'print $2'})
+    if [[ "$MEM_TOTAL" -gt "$MIN_RAM" ]]
+    then
+        return 1
+    fi
+}
+
+function set_playbook_light() {
+    PLAYBOOK_LIGHT="true"
+}
+
```

Comment:
This was specific for hornet, not sure this will apply for goshimmer (if they'll have a concept of memory profile?)